pages/docs/installation/plugins.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: "Grouparoo Plugins"
+date: "2021-02-10"
+section: "installation"
+pullQuote: "Grouparoo can be extended with Plugins to add more Apps, Sources, and Destinations."
+---
+
+Grouparoo can be extended with Plugins to add more Apps, Sources, and Destinations.
+
+## Installing a Plugin
+
+The Grouparoo command line tool is used to add a plugin to your project via `grouparoo install [package]`. You can learn more about the `grouparoo install` command [here](/docs/cli/setup). So, if you wanted to install the `@grouparoo/mysql` plugin, you would run:
+
+```bash
+grouparoo install @grouparoo/mysql # install the latest version
+```
+
+That's it!
+
+Please ensure that the versions of your plugins match the version of `@grouparoo/core` that you are using **exactly**. If you need to install a specific version of a plugin to match `@grouparoo/core`, you can:
```

Comment:
"Please ensure that the versions of your plugins match the version of `@grouparoo/core` that you are using **exactly**."
->
"Please ensure that the versions of your plugins match the version of `@grouparoo/core` that you are using **exactly** in your `package.json` file."